src/main/java/com/nomiceu/nomilabs/gregtech/prefix/LabsOrePrefix.java

```diff
@@ -0,0 +1,29 @@
+package com.nomiceu.nomilabs.gregtech.prefix;
```

Comment:
For this file:

Instead of making an empty init function, can you make the gemPerfect variable not final, and then set that in the init func instead? It would be cleaner.